tgui/packages/tgui/interfaces/CompostBin.js

```diff
@@ -0,0 +1,97 @@
+import { useBackend, useSharedState } from '../backend';
+import {
+  Button,
+  Section,
+  Box,
+  Flex,
+  Icon,
+  Collapsible,
+  NumberInput,
+  ProgressBar,
+  Dimmer,
+  LabeledList,
+} from '../components';
+import { Window } from '../layouts';
+
+
+export const CompostBin = (props, context) => {
+	const { act, data } = useBackend(context);
+	const { biomass, compost, biomass_capacity, compost_capacity} = data;
+	
+	let [vendAmount, setVendAmount] = useSharedState(context, 'vendAmount', 1);
+	
+  return (
+    <Window>
+      <Window.Content>
```

Comment:
indent _everything_ inside these two elements so that it's properly nested. children of an element should be indented more than their parent

Reply:
I just went back and tabbed everything to be(hopefully) properly idented. It's a bit wider than your suggestion but otherwise should be fine.